ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/clientlibs/site/css/textinputview.css

```diff
@@ -41,3 +41,12 @@ textarea.cmp-adaptiveform-textinput__widget{
 .cmp-adaptiveform-textinput__questionmark {
 
 }
+.cmp-adaptiveform-textinput__charcounter {
```

Comment:
we only add empty classes here, the respective CSS changes will go in theme https://github.com/adobe/aem-forms-theme-canvas

Reply:
https://github.com/adobe/aem-forms-theme-canvas/pull/47 PR for styling @sakshi-arora1